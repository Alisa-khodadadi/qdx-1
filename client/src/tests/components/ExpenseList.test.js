// import React from 'react';
// import {ExpenseList} from '../../components/ExpenseList'
// import expenses from '../fixtures/expenses'
// import {shallow} from 'enzyme'

// test('should render expense list with expense ',()=> {
//     const wrapper =  shallow(<ExpenseList expenses={expenses}/>)
//     expect(wrapper).toMatchSnapshot()
// })

// test('should render expense list without expense ',()=> {
//     const wrapper =  shallow(<ExpenseList expenses={[]}/>)
//     expect(wrapper).toMatchSnapshot()
// })