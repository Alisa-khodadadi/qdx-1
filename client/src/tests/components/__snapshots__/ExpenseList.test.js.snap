// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list with expense  1`] = `
<div>
  <ExpenseListItem
    amount={195}
    createdAt={0}
    description="gum"
    id="1"
    key="1"
  />
  <ExpenseListItem
    amount={2000000}
    createdAt={-345600000}
    description="rent"
    id="2"
    key="2"
  />
  <ExpenseListItem
    amount={4500}
    createdAt={345600000}
    description="credit card"
    id="3"
    key="3"
  />
</div>
`;

exports[`should render expense list without expense  1`] = `
<div>
  <p>
     no expenses to show
  </p>
</div>
`;
